module/localisation/BallLocalisation/src/BallModel.hpp

```diff
@@ -93,7 +93,7 @@ namespace module::localisation {
         inline int getRogueCount() const {
             return n_rogues;
         }
-        Eigen::Matrix<Scalar, 2, 1> getRogueRange() const {
+        [[nodiscard]] Eigen::Matrix<Scalar, 2, 1> get_rogue() const {
```

Comment:
whats a rogue range

Reply:
oh like rogue particles and how large of a range do we allow them to stick about?

Reply:
`rogue_range` is old terminology, not sure what it is referring to. A rogue particle can be whatever you want (i.e. sample new state from a uniform distribution spread out over the field).

Reply:
Rogue range was basically the range of values for each dimension that a rogue could be generated in. Now the rogues are just generated by the model directly